src/pages/Settings/components/ModalForm.tsx

```diff
@@ -91,17 +105,49 @@ export const SettingsModalForm: React.FC<{
       onValuesChange={onValuesChange}
     >
       <ProForm.Group>
-        <ProFormText
-          label={<FormattedMessage id="group" defaultMessage="group" />}
-          rules={[
-            {
-              required: true,
-            },
-          ]}
-          width="md"
-          name="group"
-        />{' '}
-        {/** TODO add autocomplete  */}
+        {staticData ? (
+          <ProFormText
+            label={<FormattedMessage id="group" defaultMessage="group" />}
+            rules={[
+              {
+                required: true,
+              },
+            ]}
+            width="md"
+            name="group"
+            disabled={isGlobalGroup()}
+          />
+        ) : (
+          <ProForm.Group
+            labelLayout={'twoLine'}
+            style={{
+              paddingTop: 0,
+            }}
+          >
+            <div className="ant-col ant-form-item-label">
+              <label htmlFor="group" className="ant-form-item-required" title="">
+                <FormattedMessage id="group" defaultMessage="group" />
+              </label>
+            </div>
+            <AutoComplete
+              style={{
+                width: '300px',
+              }}
+              autoFocus={true}
+              value={selectedGroup}
```

Comment:
because it's not a ProForm component 

must wrap this with 

example from src/components/ModelFields/ModalForm.tsx

```tsx
      <ProForm.Item
        name="rules"
        label={
          <span>
            <FormattedMessage id="rules" defaultMessage="rules" />{' '}
            <a
              rel="noreferrer"
              href="https://laravel.com/docs/9.x/validation#available-validation-rules"
              target="_blank"
            >
              <FormattedMessage
                id="available-validation-rules"
                defaultMessage="available-validation-rules"
              />
            </a>
          </span>
        }
        tooltip={<FormattedMessage id="available-validation-rules" />}
        valuePropName="value"
      >
        <JsonEditor />
      </ProForm.Item>
```

docs https://procomponents.ant.design/en-US/components/field-set/#proformfields 

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/214608/183424647-5aae82d4-f4c3-4179-b609-b1ef20f13c21.png">